plugins/woocommerce-blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image/edit.tsx

```diff
@@ -30,11 +29,7 @@ const Placeholder = memo( () => {
 	);
 } );
 
-export const Edit = ( {
-	context,
-}: {
-	context: ProductGalleryContext;
-} ): JSX.Element => {
+export const Edit = (): JSX.Element => {
```

Comment:
This type is redundant

```suggestion
export const Edit = () => {
```